web_m2x_options/static/src/components/base.xml

```diff
@@ -15,32 +14,17 @@
         </xpath>
     </t>
 
-    <t
+     <t
         t-name="web_m2x_options.Many2ManyTagsField"
         t-inherit="web.Many2ManyTagsField"
         t-inherit-mode="extension"
         owl="1"
     >
         <xpath expr="//Many2XAutocomplete" position="attributes">
-            <attribute name="nodeOptions">props.nodeOptions</attribute>
+            <attribute name="searchLimit">props.searchLimit</attribute>
+            <attribute name="fieldColor">props.fieldColor</attribute>
+            <attribute name="fieldColorOptions">props.fieldColorOptions</attribute>
         </xpath>
     </t>
 
-    <t t-name="web_m2x_options.Many2OneField.CreateConfirmationDialog" owl="1">
-        <Dialog title="title" size="'md'">
-            <div>
-                You are creating a new <strong t-esc="props.value" /> as a new <t
-                    t-esc="props.name"
-                />, are you sure it does not exist yet?
-            </div>
-            <t t-set-slot="footer">
-                <button class="btn btn-primary" t-on-click="onCreate">Create</button>
-                <button
-                    class="btn btn-primary"
-                    t-on-click="onCreateEdit"
-                >Create and Edit</button>
-                <button class="btn" t-on-click="() => props.close()">Discard</button>
-            </t>
-        </Dialog>
-    </t>
 </templates>
```

Comment:
Have you removed the confirm create dialog?

Reply:
@CRogos  I removed it as I don't think it was working in v15 and v16 (or at least I couldn't make this option work). Are we sure they work in previous versions? Thank you.

Reply:
You are right. Not working in v16 either.